basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -785,12 +785,85 @@ public static void addGlobalData(Site site, Properties props, Properties custom,
 		M_log.debug("before custom="+custom);
 		LTI2Util.substituteCustom(custom, lti2subst);
 		M_log.debug("after custom="+custom);
-
+		
 		// Place the custom values into the launch
 		LTI2Util.addCustomToLaunch(ltiProps, custom);
+		
+		//substitute resources params and add the rest
+		if(custom.getProperty(BasicLTIConstants.RESOURCE_LINK_ID) != null){//TODO if allowReviewService prop y future prop for TII
```

Comment:
I'm not sure if we should leave like that (and wait for a future specification of LTI tools) or add content review code for checking it, which I'm not very fond of.